packages/paste-core/components/typography/stories/index.stories.tsx

```diff
@@ -20,4 +30,13 @@ storiesOf('Components|Typography', module)
         </Paragraph>
       </>
     );
+  })
+  .add('Heading', () => {
+    const asOptions = text('as', 'h2') as asTags;
+    const headingStyleValue = select('headingStyle', headingStyleOptions, 'headingStyle50') as HeadingStyle;
+    return (
+      <Heading as={asOptions} headingStyle={headingStyleValue}>
+        I am a Very Large Heading
```

Comment:
Can you add an example with an icon in the heading or some other component?

Reply:
We may need to work with design a little on how to handle icons in the headers, i.e. size restrictions for the icons. Did you have another component in mind to use as an example here? Anchor?

Reply:
Maybe just a `Text` then?

Reply:
I added a couple more tests to `Paragraph` and `Heading`.
- italics and bold children in `Paragraph`
- italics children in `Heading`